src/Microsoft.DocAsCode.Build.Engine/MarkdownProviders/JsonTokenTreeServiceProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export("jsonTokenTree", typeof(IMarkdownServiceProvider))]
```

Comment:
so you need change it too... see the example in MSDN, better to follow the style [https://msdn.microsoft.com/en-us/library/system.componentmodel.composition.exportattribute(v=vs.110).aspx](https://msdn.microsoft.com/en-us/library/system.componentmodel.composition.exportattribute(v=vs.110).aspx)

---
In reply to: [74901666](https://github.com/dotnet/docfx/pull/608#discussion_r74901666) [](ancestors = 74901666,74901358)